ring/partition_ring_status.gohtml

```diff
@@ -15,6 +15,8 @@
                 <th>State</th>
                 <th>State updated at</th>
                 <th>Owners</th>
+                <th>Tokens</th>
```

Comment:
Since we usually [only show the Partition Ring page in ingest-storage mode](https://github.com/grafana/mimir/blob/27a7389510b3758812dfd7cb652147a22881ac58/pkg/mimir/modules.go#L378), I chose not to add any extra toggle to show or hide these values. In contexts where these values don't make sense, the entire page is usually inaccessible.